### DISCUSSION TITLE
蓝灯最好的办法是提供服务器指定的功能

### DISCUSSION THREAD

**Discussion by jicai** [getlantern/lantern-forum-cn#101] [General]
蓝灯最好的方法就是提供服务器指定的功能：付费用户可以根据自己的需要来选择想要的服务器，需要chatgpt的就用美国的服务器，需要看视频玩优秀的，就用距离大陆近些的服务器。

**Comment by MaxMKMax** [comment]
@jicai 非常好的建议. 您说的对的. 我们已经安排在计划里建立这个功能了. 也希望你们各以任意切换服务器. 

步骤一: 在这之前，您方便让我知道您现在用的是哪个版本吗？打个比方version xxx.xxx . 然后用安卓手机还是萍果手机? 或者哪款的电脑呢？您可以在留言区给我知道.

步骤二: 然后, 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

**步骤三: 当您在app提交了问题报告之后, 请回来这里留言您的log/ticket号码**

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。
